proto/proto/responses.proto

```diff
@@ -99,6 +99,11 @@ message GetTransactionInputsResponse {
 
 message SubmitProvenTransactionResponse {}
 
+message GetNotesByIdResponse {
+    // Lists Note's returned by the database
+    repeated note.Note notes = 1;
```

Comment:
It seems these are missing the details (i.e. the note's script/inputs/etc.):

https://github.com/0xPolygonMiden/miden-node/blob/0335339c6bdaf3daf07dd3750e2a19b97e47ba66/proto/proto/note.proto#L8-L15

Reply:
I don't think that this message is missing a `details` field.

This message represents getting a list of notes. Each note will have their own `details` field indeed. But the `GetNoteById` message should not return details but only a list of notes. 

If you want to see the `Note` message with a details field, then see this PR when I make the modification to the db: https://github.com/0xPolygonMiden/miden-node/pull/300